### PR TITLE
YOLOv4 quantization support

### PR DESCRIPTION
**Pull request by negvet**

### CODE REVIEW

**Review by alexsu52 — COMMENTED** [review]
Please, publish a checkpoint for the YOLOv4 quantized model.

**Review by pfinashx — APPROVED** [review]

**Comment by pfinashx** [comment]
Be aware, that we have accuracy deviation in E2E test for Yolo models, until [this](https://github.com/openvinotoolkit/open_model_zoo/pull/2219) AC PR will not be merged.

**Comment by negvet** [comment]
> Be aware, that we have accuracy deviation in E2E test for Yolo models, until [this](https://github.com/openvinotoolkit/open_model_zoo/pull/2219) AC PR will not be merged.

Yes, we know, we initiated [#2219](https://github.com/openvinotoolkit/open_model_zoo/pull/2219). Thanks

File: beta/examples/tensorflow/object_detection/README.md [hunk 1]
```diff
@@ -151,4 +151,5 @@ To export a model to the OpenVINO IR and run it using the Intel® Deep Learning
 |RetinaNet|Sparsity 50% (Magnitude)|COCO2017|33.13|[retinanet_coco_magnitude_sparsity.json](configs/sparsity/retinanet_coco_magnitude_sparsity.json)|[Link](https://storage.openvinotoolkit.org/repositories/nncf/tensorflow/models/develop/retinanet_sparsity_50.tar.gz)|
 |RetinaNet|Filter Pruning 40%|COCO2017|32.7|[retinanet_coco_pruning.json](configs/pruning/retinanet_coco_pruning.json)|[Link](https://storage.openvinotoolkit.org/repositories/nncf/tensorflow/models/develop/retinanet_pruning_40.tar.gz)|
 |YOLOv4|None|COCO2017|47.04|[yolo_v4_coco.json](configs/yolo_v4_coco.json)|[Link](https://storage.openvinotoolkit.org/repositories/nncf/tensorflow/models/develop/yolo_v4.tar.gz)|
+|YOLOv4|INT8 w:sym,per-channel a:asym,per-tensor |COCO2017|46.20|[yolo_v4_coco_int8.json](configs/quantization/yolo_v4_coco_int8.json)|[Link](https://storage.openvinotoolkit.org/repositories/nncf/tensorflow/models/develop/yolo_v4_int8_w_sym_c_a_asym_t.tar.gz)|
```
Comment (alexsu52) [inline beta/examples/tensorflow/object_detection/README.md:154]: I would like to ask you to change the checkpoint name. We have already quantized models with per-channel granularity and usually we `ch` alias for this.
```suggestion
|YOLOv4|INT8 w:sym,per-channel a:asym,per-tensor |COCO2017|46.20|[yolo_v4_coco_int8.json](configs/quantization/yolo_v4_coco_int8.json)|[Link](https://storage.openvinotoolkit.org/repositories/nncf/tensorflow/models/develop/yolo_v4_int8_w_sym_ch_a_asym_t.tar.gz)|
```

File: beta/tests/sota_checkpoints_eval.json [hunk 2]
```diff
@@ -246,7 +246,18 @@
                     "batch_per_gpu": 15,
                     "reverse_input_channels": true,
                     "scale_value": "Placeholder[255]"
-
+                },
+                "yolo_v4_int8_w_sym_c_a_asym_t": {
+                    "config":"examples/tensorflow/object_detection/configs/quantization/yolo_v4_coco_int8.json",
+                    "reference":"yolo_v4",
+                    "target":46.20,
+                    "resume":"yolo_v4_int8_w_sym_c_a_asym_t",
```
Comment (alexsu52) [inline beta/tests/sota_checkpoints_eval.json:254]:
```suggestion
                    "resume":"yolo_v4_int8_w_sym_ch_a_asym_t",
```

File: beta/tests/sota_checkpoints_eval.json [hunk 3]
```diff
@@ -246,7 +246,18 @@
...
+                    "metric_type":"mAP",
+                    "model_description":"yolo_v4_int8_w_sym_c_a_asym_t",
```
Comment (alexsu52) [inline beta/tests/sota_checkpoints_eval.json:256]:
```suggestion
                    "model_description":"yolo_v4_int8_w_sym_ch_a_asym_t",
```

File: beta/tests/tensorflow/data/ac_configs/yolo_v4_int8_w_sym_c_a_asym_t.yml [hunk 4]
```diff
@@ -0,0 +1,56 @@
+models:
```
Comment (alexsu52) [inline beta/tests/tensorflow/data/ac_configs/yolo_v4_int8_w_sym_c_a_asym_t.yml:1]: Please rename the file: yolo_v4_int8_w_sym_c_a_asym_t.yml -> yolo_v4_int8_w_sym_ch_a_asym_t.yml

**Review by alexsu52 — APPROVED** [review]

**Pull request merged**